skycoin/skywire
# Display local IP

Did you run `make format && make check`?
Yes
Fixes #389 

**Works only on Linux and MacOS systems**

This PR also contains all the commits from #685 . So if this gets merged, #685 may be closed for good.

 Changes:	
- `DefaultNetworkInterface`, `LocalNetworkInterfaceIPs`, `NetworkInterfaceIPs` were moved from `vpn` to `util/netutil`;
- `DefaultNetworkInterface` was implemented for MacOS;
- `local_ip` field was added to the response from `/api/visors/{pk}.

How to test this PR:
1. Run visor with hypervisor
2. Open network console in browser
3. Open visor page
4. Check response from `/api/visors/{pk}`. It should contain `local_ip` field with the corresponding value for you default network interface IP
